singer-io/tap-bing-ads
# Tdl 9721 missing top level breadcrumb for all streams

# Description of change
- Added top-level breadcrumb

# Manual QA steps
 - Run discover mode with and without this code and verify fields of both catalogs. 
 - There must be no missing fields.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
